interpreter/chapter.txt

```diff
@@ -372,23 +386,28 @@ c  |       (oldest)      | -> data stack: []
 k   ---------------------
 ```
 
-At this point, the interpreter is in the middle of the function call to `bar`. There are three frames on the call stack: one for the module level, one for the function `foo`, and one for `bar`. Once `bar` returns, the frame associated with it is popped off of the call stack and discarded.
```

Comment:
Huh! "out of" doesn't make sense in the context of a stack, but "popped from the call stack" would be fine.